src/openMVG/localization/LocalizationResult.cpp

```diff
@@ -92,6 +92,59 @@ double LocalizationResult::computeInliersRMSE() const
   return std::sqrt(sqrErrors.mean());
 }
 
+double LocalizationResult::computeAllRMSE() const 
+{
+  const auto& residuals = computeAllResiduals();
+  // squared residual for each point
+  const auto sqrErrors = (residuals.cwiseProduct(residuals)).colwise().sum();
+  //RMSE
+  return std::sqrt(sqrErrors.mean());
+}
+
+Vec LocalizationResult::computeReprojectionErrorPerInlier() const 
+{
+  const auto& residuals = computeInliersResiduals();
+  // squared residual for each point
+  const Vec sqrErrors = (residuals.cwiseProduct(residuals)).colwise().sum();
+  //RMSE
+  return sqrErrors.array().sqrt();
+}
+
+Vec LocalizationResult::computeReprojectionErrorPerAll() const
+{
+  const auto& residuals = computeAllResiduals();
+  // squared residual for each point
+  const Vec sqrErrors = (residuals.cwiseProduct(residuals)).colwise().sum();
+  //RMSE
+  return sqrErrors.array().sqrt();
+}
+
+std::size_t LocalizationResult::updateInliers(double threshold)
+{
+   const auto &residuals = computeReprojectionErrorPerAll();
+   auto &inliers = _matchData.vec_inliers;
+   std::cout << "Inliers before: " << inliers.size();
+   inliers.clear();
+   // at worst they could all be inliers
+   inliers.reserve(getPt2D().size());
+   
+   for(std::size_t i = 0; i < residuals.size(); ++i )
+   {
+     if(residuals[i] < threshold)
+     {
+       inliers.push_back(i);
+     }
+   }
+   std::cout << " \tafter: " << inliers.size() << std::endl;
+   _matchData.error_max = threshold;
```

Comment:
no it comes from acransac, it is actually the threshold, either used or computed by acransac